### PR TITLE
Jordan/pro 204 change founders

### PR DESCRIPTION
**Pull request by blokboy**
still catching `ONLY_OWNER` revert on failing test case for `UpdateFounderAllocationFuzz`

### CODE REVIEW

File: src/token/Token.sol [hunk 1]
```diff
@@ -321,6 +321,65 @@ contract Token is IToken, UUPS, Ownable, ReentrancyGuard, ERC721Votes, TokenStor
         return tokenRecipient[_tokenId % 100];
     }
 
+    /// @notice Update the list of allocation owners
+    /// @param newFounders the full list of founders
+    function updateFounders(IManager.FounderParams[] calldata newFounders) external onlyOwner {
```
Comment (iainnash) [inline src/token/Token.sol:326]: Same concern as above — is this update as in maybe resetFounders?

File: test/Token.t.sol [hunk 2]
```diff
@@ -440,4 +440,97 @@ contract TokenTest is NounsBuilderTest, TokenTypesV1 {
...
+    ) public {
+        deployMock();
+
+        address f1Wallet = address(0x1);
+        address f2Wallet = address(0x2);
+        address f3Wallet = address(0x3);
+
+        vm.assume(f1Percentage > 0 && f1Percentage < 100);
+        vm.assume(f2Percentage > 0 && f2Percentage < 100);
+        vm.assume(f3Percentage > 0 && f3Percentage < 100);
```
Comment (iainnash) [inline test/Token.t.sol:488]: If you pass in 1 and 2 founder ownership percentage the third one can be implicitly found through 100 - f1 - f2 = f3

Comment (tbtstl) [reply test/Token.t.sol:488]: This assumes we want `sum(f1, f2, f3) = 100` though, no? Would break the test cases for [1, 1, 1], for example

Comment (iainnash) [reply test/Token.t.sol:488]: My bad — carry on—

**Review by iainnash — APPROVED** [review]

**Pull request merged**